source/components/HexMetrics.js

```diff
@@ -77,19 +80,33 @@ export default class HexMetrics extends React.Component {
     this.props.onAddTileMouseDown(event.currentTarget.parentElement.id)
   }
 
+  _renderWarning(shouldWarn) {
+    if (!shouldWarn) return null
+    return (
+      <div id='warning'>
+        <i className='fa fa-exclamation-triangle' />
+        {` At this data resolution, some states will not be represented.
+        Consider a lower resolution.`}
```

Comment:
Curious, why embed a string in a template variable instead of just writing directly to JSX?